src/opera/pge/cslc_s1/cslc_s1_pge.py

```diff
@@ -191,6 +193,132 @@ def _json_metadata_filename(self, inter_filename):
+        the CSLC-S1 SAS. This metadata is then formatted for use with filling in
+        the ISO metadata template for the CSLC-S1 PGE.
+
+        Returns
+        -------
+        output_product_metadata : dict
+            Dictionary containing CSLC-S1 output product metadata, formatted for
+            use with the ISO metadata Jinja2 template.
+
+        """
+        # Gather the output products produced by the SAS to locate the JSON file
+        # containing the product metadata
+        output_products = self.runconfig.get_output_product_filenames()
+        json_metadata_product = None
+
+        for output_product in output_products:
+            if output_product.endswith('.json') and not output_product.endswith('.catalog.json'):
+                json_metadata_product = output_product
+                break
+        else:
+            msg = (f"Could not find the JSON metadata output product within "
+                   f"{self.runconfig.output_product_path}")
+            self.logger.critical(self.name, ErrorCode.ISO_METADATA_RENDER_FAILED, msg)
+
+        # Read the output product metadata
+        with open(json_metadata_product, 'r') as infile:
+            output_product_metadata = json.load(infile)
+
+        # Parse the burst center coordinate to conform with gml schema
+        # sample: "POINT (441737.4292702299 3877557.760490343)"
+        burst_center_str = output_product_metadata['center']
+        burst_center_pattern = r"POINT\s*\(\s*(.+)\s*\)"
+        result = re.match(burst_center_pattern, burst_center_str)
+
+        if not result:
+            msg = f'Failed to parse burst center from string "{burst_center_str}"'
+            self.logger.critical(self.name, ErrorCode.ISO_METADATA_RENDER_FAILED, msg)
+
+        output_product_metadata['burst_center'] = result.groups()[0]
+
+        # Parse the burst polygon coordinates to conform with gml
+        # sample: "POLYGON ((399015 3859970, 398975 3860000, ..., 399015 3859970))"
+        burst_polygon_str = output_product_metadata['border']
+        burst_polygon_pattern = r"POLYGON\s*\(\((.+)\)\)"
+        result = re.match(burst_polygon_pattern, burst_polygon_str)
+
+        if not result:
+            msg = f'Failed to parse burst polygon from string "{burst_polygon_str}"'
+            self.logger.critical(self.name, ErrorCode.ISO_METADATA_RENDER_FAILED, msg)
+
+        output_product_metadata['burst_polygon'] = "".join(result.groups()[0].split(','))
+
+        return output_product_metadata
+
+    def _create_custom_metadata(self):
+        """
+        Creates the "custom data" dictionary used with the ISO metadata rendering.
+
+        Custom data contains all metadata information needed for the ISO template
+        that is not found within any of the other metadata sources (such as the
+        RunConfig, output product(s), or catalog metadata).
+
+        Returns
+        -------
+        custom_data : dict
```

Comment:
small detail, but should this be "custom_metadata"?

Reply:
Nice catch, fixed